ckanext/depositar_theme/fanstatic/scss/_variables.scss

```
@@ -1,22 +1,113 @@
-// base variables
-$imagePath: '../../images';
+@use "sass:color";
+@use "sass:map";
 
+// Base variables
+$imagePath: "../../../base/images";
+$bgPath: "";
 $mainThemeColor: #00e09b;
 $secondThemeColor: #012819;
 
-$extraSmallDeviceMaxWidth: 575.98px;
-$smallDeviceMaxWidth: 767.98px;
-$mediumDeviceMaxWidth: 991.98px;
-$largeDeviceMaxWidth: 1199.98px;
-$extraLargeDeviceMaxWidth: 1399.98px;
+// Layout Variables
+$layoutFontFamily: 'Noto Sans TC', arial, 'Microsoft JhengHei', sans-serif;
```

```suggestion
$layoutFontFamily: $font-family-sans-serif;
```
Align with the [upstream](https://github.com/ckan/ckan/blob/ckan-2.10.7/ckan/public/base/scss/_variables.scss#L6).